### ISSUE TITLE
Cards: Productive type - Design feedback

### ISSUE THREAD

**Issue by andrea-gm** [carbon-design-system/ibm-cloud-cognitive#382]

**Comment by davidmenendez** [comment]
Productive card storybook can be found here http://localhost:3000/?path=/story/cloud-cognitive-canary-card--productive

currently there are only two stories- one for the default mode and one with overflow actions. everything else should be controllable from the controls.

**Comment by andrea-gm** [comment]
@davidmenendez you mentioned you're waiting on the designer, correct? Would it make sense to move this issue into "On hold" or "In progress" ?

**Comment by JohannaSuarezKoval** [comment]
Thanks again for the hard work with this component!


The productive card is a bit more complex than the expressive, so we need to have some variations. 

- The productive card needs to be a separate component from the expressive in the storybook. They are two different types and used in different contexts. Therefore the controls are different for each. For example, the expressive card's action placement doesn't change top or bottom like a Productive card. 
Can we add a "Card" folder and inside have "Expressive Card" and "Productive Card" components separated. 

- There are three different action bar placements. 
By default, all primary actions should be at the top, with the option of having a supplemental action bar at the bottom (just a ghost to the left)
The bottom bar for both the sumplemental and primary actions is optional for just special cases. 
So, can we have two variations like `Standard action bar` with a control to turn on and off the supplemental bar (the bottom one with the left ghost button).  And `Complex bottom bar` variation with all actions at the bottom. 

- The icons should be icon buttons in the productive card.

- The productive card also has similar variations in the title block: 
Title only
With label
With description
Label only

- There are also three main variations in the content that change the margin of the content block.
**Regular content:** just text
<img width="591" alt="Screen Shot 2021-03-18 at 10 57 48 AM" src="https://user-images.githubusercontent.com/5809654/111647450-e20b2900-87d8-11eb-9c5d-ec5dbc532f50.png">

**Chart content:** content starts with a chart in which there is an 8px top margin
<img width="590" alt="Screen Shot 2021-03-18 at 10 57 53 AM" src="https://user-images.githubusercontent.com/5809654/111647479-e6cfdd00-87d8-11eb-9174-1ef5cfacdf80.png">

**Stacked content:** These are clickable content blocks with dividers. Margin changes too
<img width="591" alt="Screen Shot 2021-03-18 at 10 57 58 AM" src="https://user-images.githubusercontent.com/5809654/111647530-f2230880-87d8-11eb-897c-0e2769ce1ac8.png">


- The overflow dropdow should open to the left like this... 
<img width="565" alt="Screen Shot 2021-03-18 at 10 59 38 AM" src="https://user-images.githubusercontent.com/5809654/111647737-1da5f300-87d9-11eb-9cd9-320f907b185d.png">
<img width="385" alt="Screen Shot 2021-03-18 at 12 17 38 PM" src="https://user-images.githubusercontent.com/5809654/111659899-f56fc180-87e3-11eb-8a14-2a5c1e7467a3.png">


- Thanks for adding the title size control it looks great!

- The use of media is not common in the productive card, so that we can delete those control for productive cards

- The card size control isn't working for any of the cards

Please let me know if there is any question
Thank you!

**Comment by JohannaSuarezKoval** [comment]
Hi David,
I noticing some adjustment needed in the primary actions (action icons)

- When they are at the top right they need a 8px padding the the right. The top padding also changes depending on the size of the title:

When they are at the top right and the title is in the default size there is a 12px top-padding, 
<img width="568" alt="Screen Shot 2021-04-05 at 2 57 03 PM" src="https://user-images.githubusercontent.com/5809654/113613792-b1701f80-961f-11eb-84d8-58bfaae54fd9.png">

When thy are at the top and the tittle size is large there is a 16px-padding
<img width="569" alt="Screen Shot 2021-04-05 at 3 01 22 PM" src="https://user-images.githubusercontent.com/5809654/113613871-cd73c100-961f-11eb-9671-940428296d50.png">

And when there is a label the should stay aligned to the top
<img width="500" alt="Screen Shot 2021-04-05 at 3 12 58 PM" src="https://user-images.githubusercontent.com/5809654/113615132-7a9b0900-9621-11eb-97d8-6d625d428245.png">


This variations are to have the icon always aligning to the title text

Also

- When the primary actions are at the bottom they should use the field size (40px) just like the ghost button.  In this case I suggest removing the top/bottom position control for the icon actions and just leave them available by default in the `complex bottom bar` variation using the field size icon button
<img width="573" alt="Screen Shot 2021-04-05 at 3 17 09 PM" src="https://user-images.githubusercontent.com/5809654/113615559-04e36d00-9622-11eb-98b5-3e1bf243181d.png">

**Comment by JohannaSuarezKoval** [comment]
I have a question regarding the ghost button of the productive card:
The ghost button at the bottom is actually optional as it is a supplemental action (no the primary action) of the card. Can this bottom bar with the ghost button be removed easily if it isn't need it? 
<img width="1401" alt="Screen Shot 2021-04-05 at 3 31 30 PM" src="https://user-images.githubusercontent.com/5809654/113617611-8805c280-9624-11eb-91d2-0e6d7355b416.png">

**Comment by JohannaSuarezKoval** [comment]
Question regarding clickable cards:
Productive cards can also be clickable even if they have action buttons. Has this been included? 
Please see the **action zone** section of the guidelines:
https://pages.github.ibm.com/cdai-design/pal/components/card/productive/usage#formatting

**Comment by davidmenendez** [comment]
based on slack conversations with Johanna i think we can conclude this round of design feedback and start prepping this for release review.
